cmd/aperturectl/cmd/installation/install.go

```diff
@@ -0,0 +1,60 @@
+package installation
```

Comment:
How about we call this package `install`?

Reply:
It had uninstall as well so I renamed it `installation`. I was also confused between `install` and `installation`. WDYT?

Reply:
Sounds good.